Storage/SQL/SQLiteRemoteClientsAndTabs.swift

```diff
@@ -155,6 +155,29 @@ public class SQLiteRemoteClientsAndTabs: RemoteClientsAndTabs {
         return Deferred(value: Result(success: clients))
     }
 
+    public func getTabsForClientWithGUID(guid: GUID?) -> Deferred<Result<[RemoteTab]>> {
+        var tabsSQL = "SELECT * FROM \(TableTabs) where client_guid "
```

Comment:
And consider just `let` with two branches with known-in-advance strings, one for the null case and one for the other.